modules/services/director/main.tf

```diff
@@ -27,7 +27,7 @@ resource aws_instance director {
 }
 
 resource aws_security_group director {
-  name = "director"
+  name = "edx-director"
```

Comment:
Same here.. can this have the `${environment}` in the name to prevent clashes when stage provisioning?

I'm ok with making this a variable that can be overridden for stage deployments though, if that's quicker.

Reply:
for this case I was kind of thinking on using the same director for both, but you are right :smile: , I'll make the whole name a variable

Reply:
I'd be fine with using the same director for both stage and production -- saves a VM. Just not sure how this would work with the terraform set up the way it is? E.g. we need to be able to SSH into both stage and prod VMs from the director, so the security group needs to know about both. So unless you can see a way to do that, a variable name would be best, thanks!

Reply:
The output of this module is its `security_group_id` which can be later used on the `openedx` module, so for stage you can just set the same id (copying from console for example) so the stage instance are accessible from the same director

```
module "openedx" {
  source = "git@github.com:open-craft/terraform-scripts.git//modules/services/openedx?ref=...."
  customer_name = local.customer_name
  director_security_group_id = "custom security group ID"
  environment = local.environment
  customer_domain = local.customer_domain

  route53_subdomains = local.edx_subdomains
}
```